src/popup/pages/Assets/AssetDetails.vue

```diff
@@ -73,6 +73,7 @@
             <TransactionAndTokenFilter
               :key="routeName?.toString()"
               :show-filters="showFilterBar"
+              :show-all-filter-options="!isAe && currentActiveProtocol === PROTOCOLS.aeternity"
```

Comment:
Maybe we can rename `isAe` to `isAeCoin` to avoid confusion. Right now this check seems a bit confusing to me